apps/src/templates/VersionRow.jsx

```diff
@@ -21,58 +26,101 @@ export default class VersionRow extends React.Component {
     return timestamp.toString();
   }
 
+  getUrlAttributes() {
```

Comment:
(minor) Perhaps `getQueryString()` or `getQueryParams()` would be a clearer name for this method?

Reply:
updated!